README.md

```diff
@@ -224,6 +224,14 @@ SELECT keyColumn, to_json(struct(*)) AS valueColumn FROM table
 ##### Optional Parameters:
 * **keyColumn** - key that can be used to perform de-duplication when reading 
 
+#### Periodic job
+Periodic job configuration allows to schedule a batch job to execute repeatedly every configured duration of time.
+This is an example of a periodic configuraion:
+```$xslt
```

Comment:
yaml?

Reply:
Do I need to explicitly write this is YAML?

Reply:
It's just not XSLT as you wrote in the markdown, right?

Reply:
changed to yaml